test/scripts/remove-use-strict.test.js

```
@@ -0,0 +1,107 @@
+    fs.rmSync(tempDir, { recursive: true, force: true })
+  })
+
+  function createTestFile(filename, content) {
+    const filePath = path.join(tempDir, filename)
+    fs.writeFileSync(filePath, content, "utf8")
+    return filePath
+  }
+
+  function runScript(directory) {
+    // Run the script with a custom directory
+    const scriptContent = fs.readFileSync("scripts/remove-use-strict.js", "utf8")
```

ESLint

[error] Replace `"scripts/remove-use-strict.js",·"utf8"` with `⏎······"scripts/remove-use-strict.js",⏎······"utf8"⏎····`

(prettier/prettier)